copier/config/factory.py

```diff
@@ -81,6 +97,11 @@ def make_config(
                 _metadata["commit"] = git("describe", "--tags", "--always").strip()
     # Obtain config and query data, asking the user if needed
     file_data = load_config_data(src_path, quiet=True)
+
+    if "_min_copier_version" in file_data:
+        min_copier_version = file_data["_min_copier_version"]
+        verify_minimum_version(min_copier_version)
```

Comment:
[EAFP](https://docs.python.org/3.8/glossary.html#term-eafp):

```suggestion
    try:
        verify_minimum_version(file_data["_min_copier_version"])
    except KeyError:
        pass
```